app/views/works/main.html.erb

```diff
@@ -0,0 +1,65 @@
+<main class="row">
+  <div>
+    <h3>
+      <span class="media-spotlight-header"></span>
+      Media Spotlight:
+      <% if @works.first != nil %>
```

Comment:
Watch your indentation inside of conditionals in the views - all of the view code inside should be indented